ll.tm

```diff
@@ -4992,11 +4992,15 @@ ParamAttribute -> ParamAttribute
 	| Dereferenceable
 	| ParamAttr
 	| Preallocated
+	| StructRetAttr
+	| ByRefAttr
 ;
 
+StructRetAttr -> StructRetAttr : 'sret' '(' Typ=Type ')' ;
+ByRefAttr -> ByRefAttr : 'byref' '(' Typ=Type ')' ;
```

Comment:
Add comment for `ByRefAttr`. Also for consistency, add new lines to have each non-terminal production rule on a dedicated line.

```
# ref: parseByRef
#
#   ::= byref(<type>)
ByRefAttr -> ByRefAttr
   : 'byref' '(' Typ=Type ')'
;
```

Do the same for `StructRetAttr` with ref to grammar of `parseRequiredTypeAttr`.